tools/twix/src/panels/look_at.rs

```diff
@@ -0,0 +1,220 @@
+use crate::{nao::Nao, panel::Panel, value_buffer::ValueBuffer};
+use communication::client::CyclerOutput;
+use eframe::egui::{Response, Slider, Ui, Widget};
+use log::error;
+use nalgebra::{point, Point2};
+use serde_json::Value;
+use std::{ops::RangeInclusive, str::FromStr, sync::Arc};
+use tokio::sync::mpsc;
+use types::{CameraPosition, FieldDimensions, HeadMotion, MotionCommand};
+
+use super::parameter::subscribe;
+
+#[derive(PartialEq)]
+enum LookAtType {
+    PenaltyBoxFromCenter,
+    Manual,
+}
+
+pub struct LookAtPanel {
+    nao: Arc<Nao>,
+    camera_position: Option<CameraPosition>,
+    look_at_target: Point2<f32>,
+    look_at_mode: LookAtType,
+    is_enabled: bool,
+    field_dimensions: Option<ValueBuffer>,
+    field_dimensions_update_notify_receiver: mpsc::Receiver<()>,
+    motion_command: Option<ValueBuffer>,
```

Comment:
I might have had the incorrect idea about storing them, I kinda copied the structure in the parameters panel. As you questioned here, I think the option can be gotten rid of since these paths are fixed and should be there. I went a little too overboard in catching possible error cases instead of letting it crash :D

Reply:
Fixed.